src/Common/REST/TEC/V1/Documentation/OpenAPI_Schema.php

```diff
@@ -334,36 +325,20 @@ public function validate( array $data = [] ): self {
 					$data[ $param_name ] = $param instanceof Integer ? intval( $data[ $param_name ] ) : floatval( $data[ $param_name ] );
 				}
 
-				$validator = $param->get_validator();
-				$valid     = $validator( $data[ $param_name ] );
-
-				if ( $valid ) {
-					$new_data[ $param_name ] = $param->get_sanitizer()( $data[ $param_name ] );
-					continue;
-				}
-
-				// translators: 1) is the type of the parameter, 2) is the name of the parameter.
-				$exception = new InvalidRestArgumentException( sprintf( __( '%1$s parameter `{%2$s}` is invalid.', 'the-events-calendar' ), $type, $param_name ) );
-
-				$exception->set_argument( $param_name );
-				$exception->set_internal_error_code( 'tec_rest_invalid_' . strtolower( $type ) . '_parameter' );
-
-				// translators: 1) is the type of the parameter, 2) is the name of the parameter.
-				$exception->set_details( sprintf( __( 'The %1$s parameter `{%2$s}` should be a `%3$s`, here is an example: `%4$s`.', 'the-events-calendar' ), $type, $param_name, $param->get_type(), $param->get_example() ) );
-				throw $exception;
-
+				$new_data[ $param_name ] = $data[ $param_name ];
 			}
 		}
 
-		$this->sanitized_endpoint_data = $new_data;
-
-		return $this;
-	}
-
-	/**
-	 * @inheritDoc
-	 */
-	public function sanitize(): array {
-		return $this->sanitized_endpoint_data;
+		/**
+		 * Filters the schema defined parameters.
+		 *
+		 * @since TBD
+		 *
+		 * @param array $new_data The new data.
+		 * @param OpenAPI_Schema $this The schema.
+		 *
+		 * @return array The filtered data.
+		 */
+		return (array) apply_filters( 'tec_rest_schema_filter', $new_data, $this );
```

Comment:
For this filter to be more useful, I would suggest including `$data` as a parameter, and/or `$params`.